assignments/assignment02.rb

```diff
@@ -26,10 +32,21 @@ def to_sentence(ary)
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
   # your implementation here
+  ary.reduce(:+) / ary.length
```

Comment:
You need to convert one of them to floating point: either

ary.reduce(:+).to_f / ary.length

or

ary.reduce(:+) / ary.length.to_f